### ISSUE TITLE
Author not 100.00 if all distribition have 100.00

### ISSUE THREAD

**Issue by wollmers** [cpants/www-cpants#125]
On https://cpants.cpanauthors.org/author/WOLLMERS the Kwalitee measure for all distributions is 100.00 but for the author in the right column it is 99.9.

As user I would expect the Kwalitee of the author being the average of her or his distributions on CPAN, excluding Backpan and TRIAL or  DEV distributions.

**Comment by wollmers** [comment]
It's still the case. Is there some explanation, why this happens or how the authors score is calculated?

**Comment by charsbar** [comment]
This seems to happen when some of the "latest" version of one's distributions are only available at the BackPAN. Several modules need to be fixed. Will look when I have some time.

**Comment by wollmers** [comment]
The issue isn't gone away.

**Comment by charsbar** [comment]
Thanks for the heads-up. It seems I have only pushed the fix without deploying it. It's gone now.

**Comment by wollmers** [comment]
Yes, it's gone now. Thanks a lot for your work to measure quality.
